### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by looh16**
Add a popup window for card project details on both desktop and mobile.

### CODE REVIEW

File: js/cards.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  <div class="card-tonic">
+    <img src="${card.featuredImage}" alt="${card.imageAlt
+}">
+        <div class="card-detail">
+
+            <h2>${card.title}</h2>
+            <p class="card-subtitle">${card.caption}</p>
+            <p class="card-message">
+                ${card.description}
+            </p>
```
Comment (Mre55) [inline js/cards.js:57]: Kindly, fix the problem with the image and list of technology is not as of the design provided. Please make image size as of the Figma design and the `See live` and `See source` buttons are coming out of the box edit that too.

| Your Design | Figma Design |
|-------------|---------------|
| image | image |

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @looh16,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 

- [ ] Make sure to use [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters as of the project requirement.
- [ ] The person with less commit should have to fix the suggested changes.

_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css [hunk 2]
```diff
@@ -531,13 +504,189 @@ input {
...
+  width: 778px;
+  height: fit-content;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 30px;
+  color: #344563;
+  margin-top: 8rem;
+  margin-left: 20px;
+}
```
Comment (cwaku) [inline css/styles.css:586]: - [ ] I see you tried to make changes as required by the previous reviewer but there's a little bit more to do. On the desktop version, you have the technologies list overlapping the space provided which makes some of them hidden. Kindly fix this issue to get your project approved. This is how it looks currently and it doesn't match that of the Figma design. Perhaps you should remove the `width: 778px` that you have set here.

image

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @looh16 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
 - [x] Good commit history 👍🏾 
 - [x] Linters passed 💯 
 - [x] Meaninful PR summary and Title


## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them, in order to get your project approved ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @looh16       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**